### ISSUE TITLE
DAC 9 - High Priority WCAG Level A - Keyboard Access

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#9]
<img width="421" alt="DAC_Keyboard_Access_01" src="https://user-images.githubusercontent.com/75850106/155551721-815f0279-127d-4a1a-b54a-ca482eeb42bc.png">

### Issue
Keyboard only users are unable scroll through the table beneath ‘Select an area on the map or drop-down menu to see how it compares for income deprivation measures.’ as it is not within the page tab order.

<img width="598" alt="DAC 9-1 Current code ref" src="https://user-images.githubusercontent.com/75850106/155551910-6c44bc42-2206-4d9e-822d-cbb46b8d4942.png">

### Suggested Solution
Ensure that keyboard only users can navigate onto tables. This can be achieved with the tabindex attribute with a value of 0.

<img width="602" alt="DAC 9-2 Current code ref" src="https://user-images.githubusercontent.com/75850106/155551937-6c01b0c4-35a4-4e3f-9d12-7ffa79ad7110.png">
